liquibase-core/src/test/groovy/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParserTest.groovy

```diff
@@ -792,6 +791,109 @@ not ignoreLines here
         changeLog.getChangeSets().size() == 2
     }
 
+    @Unroll
+    def parse_MultiLineRollback() throws Exception {
+        when:
+        String changeLogWithMultiLineRollback = """                
+--liquibase formatted sql
+
+--changeset eK:12345 (stripComments:false splitStatements:false endDelimiter:X runOnChange:true runAlways:true context:y dbms:mysql runInTransaction:false failOnError:false)
+create table table1 (
+    id int primary key
+);
+
+/* liquibase rollback
+ delete from table1;
+ drop table table1;
+*/
+               """.trim()
+
+        DatabaseChangeLog changeLog = new MockFormattedSqlChangeLogParser(changeLogWithMultiLineRollback).parse("asdf.sql", new ChangeLogParameters(), new JUnitResourceAccessor())
+
+        then:
+        changeLog.getChangeSets().size() == 1
+        changeLog.getChangeSets().get(0).getAuthor() == "eK"
+        changeLog.getChangeSets().get(0).getId() == "12345"
+        changeLog.getChangeSets().get(0).getRollback().getChanges().size() == 1
+        ((RawSQLChange) changeLog.getChangeSets().get(0).getRollback().getChanges().get(0)).getSql() == "delete from table1; drop table table1;"
+    }
+
+    @Unroll
+    def parse_MultiLineRollbackInBetween() throws Exception {
+        when:
+        String changeLogWithMultiLineRollback = """                
+--liquibase formatted sql
+
+--changeset eK:12345 (stripComments:false splitStatements:false endDelimiter:X runOnChange:true runAlways:true context:y dbms:mysql runInTransaction:false failOnError:false)
+create table table1 (
+    id int primary key
+);
+
+--rollback delete from table1;
+--rollback drop table table1;
+
+--changeset eK:12346 (stripComments:false splitStatements:false endDelimiter:X runOnChange:true runAlways:true context:y dbms:mysql runInTransaction:false failOnError:false)
+create table table2 (
+    id int primary key
+);
+
+/* liquibase rollback
+ delete from table2;
+ drop table table2;
+*/
+
+--ChangeSet nvoxland:3
+select (*) from table3;
+--rollback not required
```

Comment:
Should this use the new `empty` keyword?

Reply:
it should, good catch!

Reply:
Thanks for updating the test, @filipelautert !